### ISSUE TITLE
Please update to 1.20.x

### ISSUE THREAD

**Issue by MySoulcutting** [MilkBowl/Vault#919]
PLEASE!

**Comment by Geolykt** [comment]
What makes you think this is necessary?

**Comment by PalataMC** [comment]
I need vault for a plugin called RPGMARKETPLACE. My server version is 1.20 and does not activate with the latest version.
Is there any SnaapShot to download?

**Comment by Geolykt** [comment]
> I need vault for a plugin called RPGMARKETPLACE. My server version is 1.20 and does not activate with the latest version. Is there any SnaapShot to download?

Chances are you did not install an economy plugin...

**Comment by PalataMC** [comment]
if I have an economic complement. I had to remove it and put it back, and the problem was solved

**Comment by bbuny007** [comment]
so is the thread closed now ? just asking to make sure it works on 1.20.x

**Comment by Geolykt** [comment]
> so is the thread closed now ? just asking to make sure it works on 1.20.x

It generally is meaningless to ask whether Vault works on newer versions because the answer will be "yes" for the forseeable future

**Comment by Aaron2550** [comment]
Vault works just fine on any 1.14+ version

**Comment by bbuny007** [comment]
sorry for a late response but thanks and geolykt arent you a contributer? cant you close this issue?

**Comment by Geolykt** [comment]
Contributor != Collaborator

Contributor just means that I did a PR once

**Comment by tookender** [comment]
This plugin does not need an update to 1.20.x, as it already works perfectly fine on that version. This plugin will most likely not get updated for a few years, since it works fine.

**Comment by bakkbone** [comment]
It's probably worth whoever owns the listings on spigotmc.org etc, to update the supported versions then, which should help filter out some of these Issues on GH

**Comment by blue-pinapple** [comment]
For me, it stopped working as soon as I updated to 1.20.6. it just said "Obtained error when loading Vault" in the console. I'm sorry if this is the wrong place for me to post this

**Comment by Geolykt** [comment]
> For me, it stopped working as soon as I updated to 1.20.6. it just said "Obtained error when loading Vault" in the console. I'm sorry if this is the wrong place for me to post this

And, what is said error? That is about as great of an error as saying "not working"

**Comment by blue-pinapple** [comment]
Yeah I realised that, but that's literally all it said. [here is the log](https://mclo.gs/IZOEaby#L2256 ). It could well be another plugin causing the error. Would you like me to make another issue or just ignore it.

**Comment by Geolykt** [comment]
> Yeah I realised that, but that's literally all it said. [here is the log](https://mclo.gs/IZOEaby#L2256). It could well be another plugin causing the error. Would you like me to make another issue or just ignore it.

That is an error unrelated to Vault. Likely caused by `PluginLibrary` - report that they need to fix their misleading logging.

My assumption (or rather said the usual cause of these errors) is that you do not have an economy plugin installed
